elastic-stack/README.md

```diff
@@ -6,23 +6,23 @@
 
 ### DESCRIPTION
 
-Create links from SignalFx charts and alerts to any URL-addressable destination such as Kibana.
+Create links from Splunk Infrastructure Monitoring charts and alerts to any URL-addressable destination such as Kibana.
 
 The links appear in the Action menus adjacent to metric property values displayed in dashboards and alerts.
 
 The metric property values are used as query parameter and/or path variable values of the URL link.
 
 ### INSTALLATION
 
-Installation is actually configuring linking from property values on SignalFx charts and alerts to any URL-addressable destination. 
+Installation is actually configuring linking from property values on SignalFx charts and alerts to any URL-addressable destination.
```

Comment:
To install, add a data link from an Infrastructure Monitoring chart or detector to a URL. To learn more, see the section
[Clicking on a property value to follow a link](https://docs.signalfx.com/en/latest/managing/data-links.html#clicking-on-a-property-value-to-follow-a-link) in the user documentation.